src/main/java/com/mongodb/jdbc/MongoResultSet.java

```diff
@@ -813,14 +814,121 @@ public ResultSetMetaData getMetaData() throws SQLException {
 
     @Override
     public Object getObject(int columnIndex) throws SQLException {
-        throw new SQLFeatureNotSupportedException(
-                Thread.currentThread().getStackTrace()[1].toString());
+		checkBounds(columnIndex);
+		int zeroIndex = columnIndex - 1;
+		BsonValue out = current.getValues().get(columnIndex - 1).value;
```

Comment:
I guess this is the intention of the above variable?
```suggestion
		BsonValue out = current.getValues().get(zeroIndex).value;
```

Reply:
updated!